components/base/Accordion.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <div>
+    <div
+      class="card has-background-info is-hoverable"
+      :class="{ 'not-expanded': !expanded}"
+    >
+      <header class="card-header" @click="toggleCardState">
+        <p class="card-header-title">
+          {{ title }}
+        </p>
+        <a class="card-header-icon">
+          <span v-if="expanded === false" class="icon">
+            <i class="mdi mdi-plus mdi-24px" />
+          </span>
+          <span v-else class="icon">
```

Comment:
This is nitpicky and just style, but there aren't that many. I'd rather it be 

if truthy.... else ....

rather than if falsey .... else ....

just to simplify logical and relational operators

no worries though!